wradlib/tests/test_io.py

```diff
@@ -1150,5 +1151,41 @@ def test_cfradial_odim_roundtrip(self):
                                    cf3.root.time_coverage_start)
 
 
+class DemTest(unittest.TestCase):
+
+    def test_get_srtm(self):
+        targets = ["N51W000", "N52E000", "N51E001",
```

Comment:
`targets` is not used within the test. Do you want to extend this?

Reply:
Oups...